examples/Web/KafkaDependentProducer.cs

```diff
@@ -0,0 +1,30 @@
+using Confluent.Kafka;
+using System;
+using System.Threading.Tasks;
+
+
+namespace Web
+{
+    /// <summary>
+    ///     Leverages the injected KafkaClientHandle instance to allow
+    ///     Confluent.Kafka.Message{K,V}s to be produced to Kafka.
+    /// </summary>
+    public class KafkaDependentProducer<K,V>
+    {
+        IProducer<K, V> kafkaHandle;
+
+        public KafkaDependentProducer(KafkaClientHandle handle)
+        {
+            kafkaHandle = new DependentProducerBuilder<K, V>(handle.Handle).Build();
+        }
+
+        public Task ProduceAsync(string topic, Message<K, V> message)
```

Comment:
Add docstrings for these two Produce calls to instruct the user when to use which one

Reply:
good idea.